e2e/parallel/nftsFlow.test.ts

```diff
@@ -0,0 +1,203 @@
+  });
+
+  it('should display token standard', async () => {
+    const tokenStandard = await findElementByText(driver, 'ERC721');
+    expect(tokenStandard).toBeTruthy();
+  });
+
+  it('should display token contract', async () => {
+    const contractAddress = await findElementByText(driver, '0x6171…f23b');
+    expect(contractAddress).toBeTruthy();
+  });
+
+  it('should display contract creator', async () => {
+    const creatorAddress = await findElementByText(driver, 'adworld.eth');
+    expect(creatorAddress).toBeTruthy();
+  });
+
+  it('should display chain', async () => {
+    const networkName = await findElementByText(driver, 'Base');
+    expect(networkName).toBeTruthy();
+  });
+
+  it('should correctly display external link', async () => {
+    const externalLinkButton = await findElementByTestId({
+      id: 'nft-link-button-adworld.game',
+      driver,
+    });
+    expect(externalLinkButton).toBeTruthy();
+  });
+
+  it('should return back to gallery and select another NFT (ENS)', async () => {
+    await findElementByTestIdAndClick({
+      id: 'navbar-button-with-back',
+      driver,
+    });
+    await findElementByTestIdAndClick({
+      id: 'nft-thumbnail-https://lh3.googleusercontent.com/O_dtxR4ggdzoCNEAZ89s7w5eBiu8rP5TELBQcuFZyIHc-raU2qj48LSkJmEKeN64JaGa7m9X5EFYUreCCJBlx9lXW0rgjrZUL0E=s1000-1',
+      driver,
+    });
+    const ensName = findElementByText(driver, 'testmar27.eth');
+    expect(ensName).toBeTruthy();
+  });
+
+  it('should display ens registration date accurately', async () => {
+    const registeredLabel = await findElementByText(driver, 'Registered on');
+    expect(registeredLabel).toBeTruthy();
+    const registeredDate = await findElementByText(driver, 'Mar 27, 2023');
+    expect(registeredDate).toBeTruthy();
+  });
+
+  it('should display and format ens expiration date', async () => {
+    const expirationLabel = await findElementByText(driver, 'Expires in');
+    expect(expirationLabel).toBeTruthy();
+    const expirationValue = await findElementByTestId({
+      id: 'ens-expiry-value',
+      driver,
+    });
+    expect(expirationValue).toBeTruthy();
+    const expirationValueText = await expirationValue.getText();
+    expect(expirationValueText !== 'Invalid Date');
+  });
+
+  it('should be able to sort nfts alphabetically', async () => {
+    await findElementByTestIdAndClick({
+      id: 'navbar-button-with-back',
+      driver,
+    });
+    await findElementByTestIdAndClick({
+      id: 'nfts-sort-dropdown',
+      driver,
+    });
+    await findElementByTestIdAndClick({
+      id: 'nfts-sort-option-abc',
+      driver,
+    });
+    await findElementByTestIdAndClick({
+      id: 'nft-thumbnail-https://lh3.googleusercontent.com/O_dtxR4ggdzoCNEAZ89s7w5eBiu8rP5TELBQcuFZyIHc-raU2qj48LSkJmEKeN64JaGa7m9X5EFYUreCCJBlx9lXW0rgjrZUL0E=s1000-0',
+      driver,
+    });
+    const ensName = findElementByText(driver, 'testmar27.eth');
+    expect(ensName).toBeTruthy();
+  });
```

Comment:
does this properly validate the sort method was changed to alphabetical?

Reply:
The `testId` changes are the very end of the string based on collection indices. If you look at the first time we access this cell in the test, the string ends in '-1' whereas this id ends in '-0'.

Reply:
ahhhh sweet. sounds good.